adapters/handlers/rest/configure_api.go

```diff
@@ -485,25 +486,22 @@ func MakeAppState(ctx context.Context, options *swag.CommandLineOptionsGroup) *s
 
 	offloadmod, _ := appState.Modules.OffloadBackend("offload-s3")
 
-	classGetter, err := schema.NewClassGetter(
-		appState.ServerConfig.Config.SchemaRetrievalStrategy,
-		schemaParser,
-		appState.ClusterService.Raft,
-		appState.ClusterService.SchemaReader(),
+	collectionRetrievalStrategyConfigFlag := configRuntime.NewFeatureFlag(
+		configRuntime.CollectionRetrievalStrategyLDKey,
+		string(configRuntime.LeaderOnly),
+		appState.LDIntegration,
```

Comment:
I would rather have a method on `LDIntegration.NewFeatureFlag` than pass it as a argument. WDYT?

Reply:
and we could have some simple stub implementation for tests